config/environments/test.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # This is required for the Faker gem. See this issue here:
+  # https://github.com/stympy/faker/issues/266
+  config.i18n.available_locales = %w[en en_GB]
```

Comment:
most tests won't require anything other than the en_GB default. If we do move towards allowing some of DB data becoming translatable though and add UI tests we may want others.
Would a `Dir["config/locale/"].each{ ... }` approach make sense to pull in whatever languages are currently available in the codebase? If not we could just hard-code 'en_US', 'es', etc. as needed I guess

Reply:
We could do. It's really just to get the tests passing—since we won't actually run tests in different locales. So it should be fine to just have one or two values there.